### DISCUSSION TITLE
Can I use isar for more than 2k rows in 7 collections

### DISCUSSION THREAD

**Discussion by seifalmotaz** [isar/isar#284] [Help] [answered]
Hi, I I am using sqlite3 for flutter desktop app but I have seen Isar project and I am thinking to swich to it but I have two questions.

> Can Isar handle more than 2k rows in single table?

> Can Isar handle more than 5 collection that have relation ships (one to many, many to one, ...)?

**Comment by ytsingshao** [comment, marked as answer]
yes.  you can. in the backend, is libmdbx support
